### ISSUE TITLE
error: invalid instruction in arch/arm/vfp/vfphw.S

### ISSUE THREAD

**Issue by tpimh** [ClangBuiltLinux/linux#1021]
On arm32_v6 with `aspeed_g5_defconfig` with integrated-as enabled:

```
<instantiation>:1:4: error: invalid instruction
1: fmrrd r0, r1, d0
   ^
/tmp/vfphw-f9a7c9.s:715:2: note: while in macro instantiation
 .irp dr,0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15
 ^
<instantiation>:4:5: error: invalid instruction
 1: fmrrd r0, r1, d1
    ^
/tmp/vfphw-f9a7c9.s:715:2: note: while in macro instantiation
 .irp dr,0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15
 ^
...
```

**Comment by nickdesaulniers** [comment]
we have 6 open issues related to VFP instructions in ARM: https://github.com/ClangBuiltLinux/linux/issues?q=is%3Aissue+is%3Aopen+vfp
so I feel inclined to close some of these as duplicated.  cc @jcai19 

VFP IIUC was ARMv5 floating point coprocessor support.

**Comment by tpimh** [comment]
This one is potentially a duplicate of #306, need to check if the solution from there also applies to this issue.

**Comment by jcai19** [comment]
@tpimh Thanks for reporting this issue, I just tried but could not reproduce it. May I know the commands to reproduce this issue? Thanks.

The commands I used are as follows,

ARCH=arm make clean
ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- make CC=clang LLVM_IAS=1 aspeed_g5_defconfig
ARCH=arm CROSS_COMPILE=arm-linux-gnueabihf- make -j32 CC=clang LLVM_IAS=1

I did see some errors but none of them were the same as the reported.
